import-service/src/handlers/importFileParser.ts

```diff
@@ -0,0 +1,41 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import * as csv from "csv-parser";
+import * as AWS from "aws-sdk";
+
+const BUCKET = process.env.BUCKET;
+
+export const importFileParser: APIGatewayProxyHandler = (event) => {
```

Comment:
Hi @Stanislav-Berezovsky ! I found some issues here with this lambda function:
1) It's not `async` as needed for first additional task;
2) You're supposed to _delete_ uploaded `.cvs` file after copying it to `parsed` folder;
3) Lastly, according to task, it should return a _correct HTTP response code_.

Can you please fix this moments?